### PR TITLE
docs: Add docs on iframe sandboxing

### PR DESCRIPTION
**Pull request by sharat87**

### CODE REVIEW

**Review by mohanarpit — APPROVED** [review]

File: website/docs/product/security.md [hunk 1]
```diff
@@ -43,6 +43,18 @@ You can add **secrets** to **APIs** or **datasource configurations** as they are
 * Appsmith does not allow some actions like `Fetch`. You cannot call an external API directly from the JavaScript code. However, you can add an API on Appsmith and use it to request, read data, or manipulate the response from the external API.
 * You should not store sensitive information using a `storeValue` function because the data is stored in the browser's local storage and can be read.
 
+## Sandboxing iframe widgets
+
+The `iframe` widget on older versions of Appsmith are vulnerable to XSS attacks. This was fixed in v1.8.6 of Appsmith. In order to for this fix is applied on your Appsmith instance, please ensure you have the following environment variable set in your `stacks/configuration/docker.env` file:
```
Comment (mohanarpit) [inline website/docs/product/security.md:48]: > In order to for this fix is applied on your Appsmith instance

Nit: This sentence is weird. Can we consider rewording this?

**Comment by mohanarpit** [comment]
Base content LGTM. I'd request to fix the linter checks before merging.

File: website/docs/product/security.md (hunk 1 above)
Comment (sharat87) [reply website/docs/product/security.md:48]: Yeah just noticed it. Already on it. 🙂

File: website/docs/reference/widgets/iframe.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ description: Iframe widget is used to display iframes in your app.
 
 ![Use iframes to embed other webpages within your applications](</img/as_iframe_cover.png>)
 
+:::info
+**Heads-up**: please ensure you have your iframe widgets sandboxed. See [Security](/product/security#sandboxing-iframe-widgets) for more information.
```
Comment (mohanarpit) [inline website/docs/reference/widgets/iframe.md:10]:
```suggestion
**Heads-up**: please ensure you have your iframe widgets sandboxed. By default, this is enabled for all Appsmith instances. See [Security](/product/security#sandboxing-iframe-widgets) for more information.
```

**Review by danciaclara — CHANGES_REQUESTED** [review]

**Review by danciaclara — APPROVED** [review]

**Pull request merged**